frontend/src/components/BestDesignQuality/index.jsx

```diff
@@ -24,7 +25,11 @@ const BestDesignQuality = () => {
     });
 
     return (
-      <section className="text-c300 mt-32 bg-c400">
+      <Container
+        margin={CONTAINER_OPTIONS.MARGIN.TOP}
+        isBgGray
+        classNames="text-c300"
+      >
         <div className="container grid grid-cols-1 lg:grid-cols-2 lg:gap-16 justify-center">
```

Comment:
remove the container class name, it's already included at the reusable container